webcrypto/tests/wpt-patches/WebCryptoAPI__digest__digest.https.any.js.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/WebCryptoAPI/digest/digest.https.any.js b/WebCryptoAPI/digest/digest.https.any.js
+index 379d9311f..949b1669f 100644
+--- a/WebCryptoAPI/digest/digest.https.any.js
++++ b/WebCryptoAPI/digest/digest.https.any.js
+@@ -119,9 +119,6 @@
+     });
+ 
+ 
+-    done();
+-
+-
```

Comment:
This seems very strange ... Can we maybe have comments in the places where we do strange changes :)